x-pack/plugins/security_solution/public/management/components/package_action_item/package_action_formatter.ts

```diff
@@ -101,9 +101,12 @@ export class PackageActionFormatter {
     code: number,
     status: FleetServerAgentComponentStatus
   ): PackageActions {
-    if (code === ENDPOINT_ERROR_CODES.ES_CONNECTION_ERROR) {
+    if (
+      code === ENDPOINT_ERROR_CODES.ES_CONNECTION_ERROR ||
+      code === ENDPOINT_ERROR_CODES.OUTPUT_SERVER_ERROR
```

Comment:
@dasansol92 So here the error is formatted to `es_connection` for both ES connection errors and output server errors. That's why we see the duplicate error for ES connection I'm guessing. Also in the screenshot I see the API returns two almost identical errors with error code `-273`, although one has a `type: input` and the other `type: output`. 🤔

Reply:
Yeah, exactly. My concern (and question) was about if this is a testing data and it's expected or not 🙂

Reply:
I was just verbalizing what you asked 😉

Reply:
@ashokaditya @dasansol92 

These errors come from the Endpoint codebase.  It's a private repo, so cannot share here, but can offline.  There are two errors, -272 and -273 which have to do with failing to reach the output server and/or ES.  I wanted to catch both of them here.  I tested and validated this with a real Endpoint which is how I caught the original bug.

Also, the error is reported twice in both input and output, which is why you see it in two places in the UI.  I don't want to suppress either error right now so that we don't miss anything that comes through.  One thought I had was to dedupe the errors in the final array right before displaying in the UI.  This would ensure we don't see errors twice and and that we don't miss any unique errors (either in input or output).  However, I'd like to do this in another PR and maybe not one that I backport to 8.7 so that I don't change the established logic too much without more testing.  Let me know your thoughts

Reply:
Makes sense @kevinlog. 🚢 it